### PR TITLE
Miscellanous code cleanups/fixes

### PR DESCRIPTION
**Pull request by Blaisorblade**
This PR is a grab bag of changes for review—probably shouldn't be in a single PR. Most should be obvious, some aren't.
This happens to include #421 for now, but that should be kept separate.

To split out:
- commits dealing with `ssFile`.
- deletions of dead code could be combined.

### CODE REVIEW

File: app/src/main/java/com/kamron/pogoiv/MainActivity.java [hunk 1]
```diff
@@ -402,21 +401,4 @@ private void startScreenService() {
...
-            cursor.moveToFirst();
-            return cursor.getString(column_index);
-        } finally {
-            if (cursor != null) {
-                cursor.close();
-            }
-        }
-    }
-
 }
```
Comment (sarav) [inline app/src/main/java/com/kamron/pogoiv/MainActivity.java]: Oh thanks! I thought I deleted it when I moved all the stuff to ScreenshotHelper.

File: app/src/main/java/com/kamron/pogoiv/ScreenShotHelper.java [hunk 2]
```diff
@@ -24,7 +24,7 @@
     private ContentObserver mediaObserver;
     private ContentResolver contentResolver;
 
-    private String getRealPathFromUri(Context context, Uri contentUri) {
```
Comment (sarav) [inline app/src/main/java/com/kamron/pogoiv/ScreenShotHelper.java]: Looks like I missed a couple of things I meant to delete. :-/

**Comment by nahojjjen** [comment]
I have not tested if the bugs you're referring to are fixed, but I'll just say that I tested the changes, and it seems to be working on my device, so nothing obvious has been broken. So lgtm. 👍

**Comment by Blaisorblade** [comment]
I've added the misc fixes on master directly, the rest is in #421 and #434. Closing this one.

**Pull request closed without merging**